src/IIIFPresentation/API.Tests/Integration/Infrastructure/TestAuthenticator.cs

```diff
@@ -11,7 +11,7 @@ public class TestAuthenticator : IAuthenticator
 {
     private const string AuthHeader = "Authorization";
     
-    public async Task<AuthResult> ValidateRequest(HttpRequest request)
+    public async Task<AuthResult> ValidateRequest(HttpRequest request, CancellationToken cancellationToken = default)

```

Comment:
is this cancellation token being used?

Reply:
Updated actual implementation of `IAuthenticator` to use token but `TestAuthenticator` has no need to use